SoftLayer/managers/user.py

```diff
@@ -267,6 +268,60 @@ def add_api_authentication_key(self, user_id):
         """
         return self.user_service.addApiAuthenticationKey(id=user_id)
 
+    def vpn_manual(self, user_id, value):
+        """Enable or disable the manual config of subnets.
+
+        :param int user_id: User to edit.
+        :param bool value: Value for vpnManualConfig flag.
+        """
+        user_object = {'vpnManualConfig': value}
+        return self.edit_user(user_id, user_object)
+
+    def vpn_subnet_add(self, user_id, subnet_ids):
+        """Add subnets for a user.
+
+        :param int user_id: User to edit.
+        :param list subnet_ids: list of subnet Ids.
+        """
+        overrides = [{"userId": user_id, "subnetId": subnet_id} for subnet_id in subnet_ids]
+        return_value = self.override_service.createObjects(overrides)
+        return return_value and self.user_service.updateVpnUser(id=user_id)
```

Comment:
This syntax feels weird to me, and im not sure using AND with a boolean and an object will work great.

Something like this is more clear.
```
        overrides = [{"userId": user_id, "subnetId": subnet_id} for subnet_id in subnet_ids]
        return_value = self.override_service.createObjects(overrides)
        update_success = self.user_service.updateVpnUser(id=user_id)
        if not update_success :
            raise exceptions.SoftLayerAPIError("Overrides created, but unable to update VPN user")
        return return_value 
```

Reply:
Syntax updated